QMLComponents/models/term.h

```diff
@@ -34,25 +34,27 @@
 class Term
 {
 public:
-	Term(const std::vector<std::string> components, const columnTypeVec& types	= { columnType::unknown }	);
-	Term(const std::string				component, columnType type				= columnType::unknown		);
-	Term(const QStringList				components, const columnTypeVec& types	= { columnType::unknown }	);
-	Term(const QString					component, columnType type				= columnType::unknown		);
-
-	const QStringList			& components()	const;
-	const QString				& asQString()	const;
-
-	std::vector<std::string>	scomponents()	const;
-	std::string					asString()		const;
+	Term(const std::vector<std::string> &	components, const columnTypeVec	&	types	= { columnType::unknown }	);
+	Term(const std::string				&	component,	const columnType		type	= columnType::unknown		);
+	Term(const QStringList				&	components, const columnTypeVec	&	types	= { columnType::unknown }	);
+	Term(const QString					&	component,	const columnType		type	= columnType::unknown		);
+	Term(const QString					&	value,		const QString		&	label,	const QString	& info = QString());
+
+	const QStringList		&	components()			const;
+	const QString			&	label()					const { return _label;	}
+	const QString			&	value()					const { return _value;	}
```

Comment:
I see you do `fq()` a lot on label and value, maybe it wouldve been better to store them as std:string hahaha

Reply:
I think that are still more functions that need a QString...